src/common/search.js

```diff
@@ -17,10 +16,13 @@ async function attempt(scan, loc, doc) {
 }
 
 function aggregate(results) {
-  return results.reduce(({ tickets, errors }, result) => ({
-    errors: errors.concat(result.error ? [result.error] : []),
-    tickets: tickets.concat(result.tickets || []),
-  }), { tickets: [], errors: [] });
+  return results.reduce(
```

Comment:
Note: this is only a side-effect of the linter having fun after line 32 got shorter 😉